### PR TITLE
[buteo-sync-plugins] Use libcontacts Seaside helpers for vCard

### PR DESCRIPTION
**Pull request by chriadam**
This commit reduces code duplication, and adds support for importing
PHOTO fields as contact avatars via Bluetooth / SyncML.

### CODE REVIEW

**Comment by chriadam** [comment]
Requires https://github.com/nemomobile/libcontacts/pull/97

**Comment by chriadam** [comment]
@matthewvogt @blammit @VDVsx  New one

File: storageplugins/hcontacts/ContactsBackend.cpp [hunk 1]
```diff
@@ -404,53 +445,38 @@ void ContactsBackend::prepareContactSave(QList<QContact> *contactList)
...
-    //QVersitReader needs LF/CRLF/CR between successive vcard's in the list,
-    //CRLF didn't work though.
-    const QString LF = "\n";
-
-    foreach ( const QString& vcard, aVCardList )
-    {
-        byteArray.append(vcard.toUtf8());
-        byteArray.append(LF.toUtf8());
+    QString crlfSequence;
+    crlfSequence += QChar::CarriageReturn + QChar::LineFeed;
```
Comment (chriadam) [inline storageplugins/hcontacts/ContactsBackend.cpp]: this doesn't work ;-)   It applies the addition as a sum rather than append

Comment (matthewvogt) [inline storageplugins/hcontacts/ContactsBackend.cpp]: "+= QChar(QChar::CarriageReturn) + QChar(QChar::LineFeed)"?

**Comment by chriadam** [comment]
Superceded by https://github.com/nemomobile/buteo-sync-plugins/pull/41

**Pull request closed without merging**